package/devel/gdb/Makefile

```diff
@@ -61,10 +61,7 @@ CONFIGURE_VARS+= \
 	ac_cv_search_tgetent="$(TARGET_LDFLAGS) -lncurses -lreadline"
 
 define Build/Compile
-	+$(MAKE) $(PKG_JOBS) -C $(PKG_BUILD_DIR) \
-		DESTDIR="$(PKG_INSTALL_DIR)" \
-		CPPFLAGS="$(TARGET_CPPFLAGS)" \
-		all
+$(call Build/Compile/Default,all)
```

Comment:
I don't think the 'all' is even necessary. Did you try to remove the Build/Compile section entirely?